recipes/cppcheck/all/conanfile.py

```diff
@@ -0,0 +1,71 @@
+"""Conan recipe package for cppcheck
+"""
+import os
+from conans import ConanFile, CMake, tools
+
+
+class CppcheckConan(ConanFile):
+    name = "cppcheck"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/danmar/cppcheck"
+    topics = ("Cpp Check", "static analyzer")
+    description = "Cppcheck is an analysis tool for C/C++ code."
+    license = "GPL-3.0-or-later"
+    generators = "cmake"
```

Comment:
```suggestion
    generators = "cmake", "cmake_paths", "cmake_find_package"
```
So that you don't have to do `set(Z3_LIBRARIES CONAN_PKG::z3)` in CMakeLists.txt